src/technologies/h.json

```
@@ -1788,11 +1788,39 @@
     "scriptSrc": "/v1/lst/universal-script\\?ph\\=",
     "website": "https://hyros.com"
   },
+  "Hyva Checkout": {
+    "cats": [
+      6
+    ],
+    "description": "A Checkout built for Hyvä Themes using Magewire, Alpine.js and Tailwind CSS.",
+    "js": {
+      "hyvaCheckout": ""
+    },
+    "headers": {
+      "X-Built-With": "^Hyva Checkout$"
+    },
+    "icon": "Hyva Themes.svg",
+    "implies": [
+      "Magento\\;version:2",
```

I don't think the version should be here. As this is just a list of technologies.

```suggestion
      "Magento",
```

---

Ah, so it doesn't attribute any Hyvä site to Magento 2 stats automatically when it's implied?

As Hyvä likely prevents Magento sites to be recognized as such, I thought this would make sure Hyvä sites are added to the general Magento 2 dataset.

Or does it do that, but not consider the version and just adds it to "magento"?

---

Yes it does attribute any Hyvä site to Magento stats automatically when it's implied, even if Magento couldn't be detected otherwise.

But I don't think it supports setting the version in the `implies` section. But maybe it does since the test below HAS detected it as Magento 2. So I guess I'm wrong here!